__tests__/meta.test.js

```diff
@@ -0,0 +1,65 @@
+const db = require("../db/connection");
+const seed = require("../db/seeds/seed");
+const data = require("../db/data/test-data/index");
+const request = require("supertest");
+const app = require("../API/app");
+
+beforeEach(() => {
+  return seed(data);
+});
+
+afterAll(() => {
+  db.end();
+});
+
+describe("GET /api", () => {
+  test("Endpoint should return a JSON Object", () => {
+    return request(app)
+      .get("/api")
+      .expect(200)
+      .then(({ body }) => {
+        const { endpoints } = body;
+        console.log(endpoints);
+        expect(typeof endpoints).toEqual("string");
+      });
+  });
+  test("JSON should parse to an object populated with endpoint objects", () => {
+    return request(app)
+      .get("/api")
+      .expect(200)
+      .then(({ body }) => {
+        const endpoints = JSON.parse(body.endpoints);
+
+        expect(endpoints.constructor).toEqual(Object);
+
+        const endpointObjs = Object.values(endpoints);
+
+        expect(endpointObjs.length > 0).toEqual(true);
+
+        endpointObjs.forEach((endpoint) => {
+          expect(endpoint.constructor).toEqual(Object);
+        });
+      });
+  });
+  test("Each endpoint object should contain the expected keys, in the expected format", () => {
+    return request(app)
+      .get("/api")
+      .expect(200)
+      .then(({ body }) => {
+        const endpoints = Object.values(JSON.parse(body.endpoints));
+
+        expect(endpoints.length > 0).toEqual(true);
+
+        endpoints.forEach((endpoint) => {
+          expect(endpoint).toEqual(
+            expect.objectContaining({
+              description: expect.any(String),
+              queries: expect.any(Array),
+              exampleResponse: expect.any(Object),
+              reqBodyFormat: expect.any(Object),
+            })
+          );
+        });
+      });
+  });
```

Comment:
These tests work and are pretty flexible  - you could achieve this tests in a different way by checking that the JSON that comes back from the request matches the contents of the `endpoints.json` file in this repo 🔍